### DISCUSSION TITLE
YouTube causing “caller does not have permission” error in movie info

### DISCUSSION THREAD

**Discussion by TimboZero** [jurialmunkey/skin.arctic.horizon.2#1038] [Discussion]
If I select a movie (either in an addon or in my collection on hdd) select information page for that movie (this doesn’t cause error) but when I press down to get cast etc, an error pops up by the clock/date saying as above. If I press left and select trailer this works fine. I do not know what other YouTube addon ‘calls’ the skin is making during the display cast etc info panel

**Comment by Nucrea** [comment]
> If I select a movie (either in an addon or in my collection on hdd) select information page for that movie (this doesn’t cause error) but when I press down to get cast etc, an error pops up by the clock/date saying as above. If I press left and select trailer this works fine. I do not know what other YouTube addon ‘calls’ the skin is making during the display cast etc info panel

Had the same issue with default skin, so maybe it's not a problem of arctic skin. Fixed it by adding youtube api keys in app's settings

**Reply by TimboZero** [reply]
No, the YouTube addon is working fine, press left on a film/show and the trailer can play without issue, as can YouTube use directly.
It seems to be part of the info screen, more specifically the Videos subsection under Cast list. The titles for both appear, the error occurs and the Video title disappears. This is either a problem in the skin or the tmdbhelper addon the skin uses.
I suspect (but am bot sure) old YouTube calls https://www.youtube.com/watch?v=uniqueid  instead of https://www.youtube.com/embed/uniqueId might be why, could be the skin or the tmdbhelper addon causing it.

**Comment by TimboZero** [comment]
The latest YouTube addon now works with personal api keys again, the videos u Ser cast will find nothing and are removed for extra info but the error is gone now.
